test/will.js

```diff
@@ -113,6 +113,24 @@ test('delivers old will in case of a crash', function (t) {
   })
 })
 
+test('delete old broker', function (t) {
+  t.plan(1)
+
+  var heartbeatInterval = 100
+  const broker = aedes({
+    heartbeatInterval: heartbeatInterval
+  })
+  t.tearDown(broker.close.bind(broker))
+
+  var brokerId = 'dummyBroker'
+
+  broker.brokers[brokerId] = Date.now() - heartbeatInterval * 3.5
+
+  setTimeout(() => {
+    t.equal(broker.brokers[brokerId], undefined, 'Broker deleted')
+  }, heartbeatInterval * 4)
```

Comment:
can we use http://npm.im/lolex to test this?

Reply:
Why? BTW lolex is deprecated in favor of https://github.com/sinonjs/fake-timers

Reply:
timers are highly unreliable in CI.

Reply:
I don't really understand how they work. I have tried to simply add 

```js
require("@sinonjs/fake-timers").install()
```

To `test/helper.js` but seems timers are not working. I have also tried with other methods but I can't make them work